bookstore/client/store_client.py

```diff
@@ -7,37 +7,38 @@
 class BookstoreClient(CurrentNotebookClient):
     """Represents a bookstore client that corresponds to the active nb client"""
 
-    def __init__(self, s3_bucket=""):
+    def __init__(self, s3_bucket=None):
         if s3_bucket:
             self.default_bucket = s3_bucket
         super().__init__()
 
     @property
     def publish_endpoint(self):
+        # TODO: refactor to /api/bookstore/publish/ ?
         api_endpoint = "/api/bookstore/published/"
         return f"{self.url}{api_endpoint}"
 
     def publish(self, path=None):
         """Publish notebook to bookstore"""
         if path is None:
             path = self.session.path
-        nb_json = self.get_contents(self.session.path)['content']
+        nb_json = self.get_contents(path)['content']
         json_body = {"type": "notebook", "content": nb_json}
 
-        target_url = f"{self.publish_endpoint}{self.session.path}"
+        target_url = f"{self.publish_endpoint}{path}"
 
-        resp = self.req_session.put(target_url, json=json_body)
-        return resp
+        response = self.req_session.put(target_url, json=json_body)
+        return response
 
     @property
     def clone_endpoint(self):
+        # TODO: refactor to /api/bookstore/clone/ ?
```

Comment:
Same comments as above around "published"… this is specifically a request from @rgbkrk to have this be more REST-ful in its naming. Rather than "clone" the action, this is intended to be a reference to a location "cloned" that refers to the notebooks that are there.

I find this even more confusing than the `/published` name, but, again, I don't want to break from this approach without getting @rgbkrk's sign off.

Reply:
Let's change it, there was general consensus from folks that the action routes shouldn't be past-tensed, but retrieval routes can be nouns. If it's generating the object let's leave it a present tense, it it's grabbing older generated objects then the past tense or pluralized nouns is probably correct.

Reply:
Ok, a change PR is underway.